src/analyses/singleThreadedLifter.ml

```diff
@@ -0,0 +1,59 @@
+(** This lifter transforms any analysis into a single threaded analysis by returning top when the code might be multi-threaded.
```

Comment:
This sounds backwards to me: it takes an analysis which only works/is sound for single-threaded code and allows it to run on multi-threaded programs (by being top).
Or is the purpose to actually take a fine multi-threaded analysis and make it less precise?